docs/user/troubleshooting/plasma.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Plasma Troubleshooting
+summary: Troubleshooting Issues With the Plasma Desktop
+---
+
+# Plasma Desktop Troubleshooting
+
+This contains fixes for various issues you might encounter with the Plasma desktop environment.
+
+## Desktop no longer responds to clicks, cursor stuck in "move" mode (cross)
+
+This is often caused by a Firefox bug. It can be triggered by various things, like moving Firefox windows.
+
+**Symptoms:**
+- Can't click on anything on the desktop.
+- Bringing up windows with keyboard shortcuts (ex: Super for the app menu, F9 for a drop down terminal) works, but they don't get focused. You can't interact with them.
+- Mouse cursor is stuck in move mode, it looks like a cross.
+
```

Comment:
I disagree. I think this would just clutter the TOC with needless headings. People can already go to each problem header, which they should read to make sure their situation matches the symptoms. 

Practically, the solutions start just a few lines before the problem summary header, so it's in view anyway.